types/urijs/index.d.ts

```diff
@@ -1,45 +1,38 @@
-// Type definitions for URI.js 1.19.1
-// Project: https://github.com/medialize/URI.js
+// Type definitions for URI.js 1.19
+// Project: http://medialize.github.io/uri.js
```

Comment:
Actually this URL is case-sensitive, only when you change the `uri` to upper case you can go to the document site

Reply:
done